package.json

```diff
@@ -8,7 +8,7 @@
   "mainSrc": "src/lib/index.js",
   "scripts": {
     "analyze": "source-map-explorer 'dist/*.js'",
-    "build": "rimraf dist && NODE_ENV=production babel src/lib --out-dir dist --copy-files && rimraf dist/**/*.test.js && rimraf dist/**/__snapshots__",
+    "build": "rimraf dist && cross-env NODE_ENV=production babel src/lib --out-dir dist --copy-files && rimraf dist/**/*.test.js && rimraf dist/**/__snapshots__",
```

Comment:
We can, but I don't think it's an issue. `cross-env` is a widely use stable dependency to handle this use case.
And because it's only use during the build, it doesn't have an impact on the size on the build itself.

Reply:
A big reason why you use a library in the first place is to get the benefits of the code without having to worry about all the details. The drawback is that you’ve completely tied your code to these dependencies that you don’t own and don’t control. Instead, you could have used an existing babel option.